analytics.go

```diff
@@ -143,7 +142,7 @@ func New(key string) *Client {
 		now:      time.Now,
 		uid:      uid,
 	}
-
+	go c.loop()
```

Comment:
Ref: https://github.com/segmentio/analytics-go/pull/41

My moving this here, we essentially don't give users a way to configure https://github.com/segmentio/analytics-go/blob/close-client-twice/analytics.go#L311.

Once we have the config stuff, though, this should be fine 👍